extras/dgoss/dgoss

```diff
@@ -20,6 +20,8 @@ cleanup() {
     set +e
     { kill "$log_pid" && wait "$log_pid"; } 2> /dev/null
     if [ -n "$CONTAINER_LOG_OUTPUT" ]; then
+        info "Stopping container and copy log"
+        $CONTAINER_RUNTIME stop -t 1 "$id" > /dev/null
```

Comment:
Does the -t flag work consistently with podman or docker?

Reply:
Yes.

- Podman: https://docs.podman.io/en/stable/markdown/podman-stop.1.html#time-t-seconds
- Docker: https://docs.docker.com/reference/cli/docker/container/stop/#time